firewood/src/merkle/node.rs

```diff
@@ -722,6 +722,7 @@ pub trait BinarySerde {
     ) -> Result<T, Self::DeserializeError>;
 }
 
+#[derive(Default)]
```

Comment:
nit: I'm not sure why this was added here.

Q: Will all new codecs require `Default`? If so, we might want to be more explicit elsewhere.

Reply:
no, added now only for test cases usage.